### PR TITLE
Change app icon on Android

### PR DESCRIPTION
**Pull request by iadmir**
### First time contributor checklist

- [x] I have read [how to contribute](https://github.com/WhisperSystems/Signal-Android/blob/master/CONTRIBUTING.md) to this project
- [x] I have signed the [Contributor License Agreement](https://whispersystems.org/cla/)

### Contributor checklist

- [x] I am following the [Code Style Guidelines](https://github.com/WhisperSystems/Signal-Android/wiki/Code-Style-Guidelines)

- [x] My contribution is fully baked and ready to be merged as is
- [x] I ensure that all the open issues my contribution fixes are mentioned in the commit message of my first commit using the `Fixes #1234` [syntax](https://help.github.com/articles/closing-issues-via-commit-messages/)
- [x] I have made the choice whether I want the [BitHub reward](https://github.com/WhisperSystems/Signal-Android/wiki/BitHub-Rewards) or not by omitting or adding the word `FREEBIE` in the commit message of my first commit

----------

### CODE REVIEW

**Comment by sweeper3000** [comment]
@iadmir please sign the CLA. The devs need it signed before they can merge.

**Comment by moxie0** [comment]
It looks like this only updates the hdpi assets?

**Comment by Stanzilla** [comment]
It's also missing the roundIcons for Pixel devices the original issue was about https://github.com/WhisperSystems/Signal-Android/issues/5936

**Comment by p4nci** [comment]
@iadmir please read [Googles Guidelines for Action Bar Icons](https://developer.android.com/guide/practices/ui_guidelines/icon_design_action_bar.html#style11):
- Action Bar icons should not look three-dimensional
- Icons should have a transparent background

IMO: for actionbar icon we don't need a blue background (see also [the current one icon](https://raw.githubusercontent.com/WhisperSystems/Signal-Android/3.28.0/res/drawable-hdpi/actionbar_icon_holo_dark.png) - v3.28.0)

**Comment by iadmir** [comment]
Other assets added, also round icon for pixel launcher

**Comment by sweeper3000** [comment]
@iadmir in order for the pixel launcher to load the round icon, you need to add `android:roundIcon` in the manifest pointing to the round icon. 

[From Android Developers](https://developer.android.com/about/versions/nougat/android-7.1.html#circular-icons)

**Comment by p4nci** [comment]
@iadmir 
a) [drawable-xxhdpi/actionbar_icon_holo_dark](https://github.com/iadmir/Signal-Android/blob/3603fc47cf7f740b5287688cdad313e2e4db25f3/res/drawable-xxhdpi/actionbar_icon_holo_dark.png) have still (blue) background
b) drawable-xxhdpi/actionbar_icon_holo_light is missed

File: AndroidManifest.xml [hunk 1]
```diff
@@ -91,6 +91,7 @@
 
     <application android:name=".ApplicationContext"
                  android:icon="@drawable/icon"
+                 android:roundIcon="@drawable/icon-pixel-launcher"
```
Comment (moxie0) [inline AndroidManifest.xml]: icon_pixel_launcher (or at least that's the name of the file)

**Comment by moxie0** [comment]
Just ran this branch, it looks like the notification icons haven't been updated. Have you tested these changes?

**Comment by iadmir** [comment]
Yes, I noticed the problem, I will open a new PR after corrections.

**Pull request closed without merging**